prover/Makefile

```diff
@@ -57,7 +57,7 @@ zkevm/define/define.go: corset
 ## Generate the setup for the execution prover (to be run with S3 access)
 ##
 setup: bin/prover
-	bin/prover setup --config ./config/config-integration-full.toml --dict ./lib/compressor/compressor_dict.bin --assets-dir ./prover-assets
+	bin/prover setup --config ./config/config-sepolia-full.toml --dict ./lib/compressor/compressor_dict.bin --assets-dir ./prover-assets
```

Comment:
We will have to pay attention to which config we are using. Because for mainnet we don't want to use the configuration of sepolia